kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/OpenAPIConfig.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server.api
+
+import io.swagger.v3.jaxrs2.integration.resources.OpenApiResource
+import io.swagger.v3.oas.annotations.OpenAPIDefinition
+import io.swagger.v3.oas.annotations.info.{Contact, Info, License}
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.glassfish.jersey.server.ResourceConfig
+
+@OpenAPIDefinition(
+  info = new Info(
+    title = "Apache Kyuubi RESTful Api Documentation",
+    version = "1.4.0",
+    description = "Apache Kyuubi RESTful Api Documentation",
```

Comment:
From previous discussion, I think we reach an agreement that use term **REST API** in all place.